src/Aws/CloudFront/Core.hs

```diff
@@ -146,7 +157,7 @@ cloudFrontSignQuery query _conf sigData = SignedQuery {
     action = cloudFrontQueryAction query
     path = apiVersion:(cloudFrontQueryPathSegments query)
     apiVersion = "2014-11-06"
-    host = "cloudfront.amazonaws.com"
+    host = T.encodeUtf8 $ _cloudFrontConfiguration_host cfConf
```

Comment:
If the call site takes a ByteString, I'm curious why you're talking a Text for the host. I don't think it's the wrong call, just want to know the reasoning.

Reply:
I returned ByteString. Host name is loaded from Aeson. I think Aeson works smoother with Text.  I will do conversion in the app.